src/tests/cwrap/group

```diff
@@ -1,2 +1,3 @@
+root:x:0:
```

Comment:
Why do you need to change the `group` and `passwd` cwrap files if you aren't changing those tests?

Reply:
we try to resolve uid/gid of SSSD_USER now and this can be compiled also with value "root" (default for Fedora). The tests  does not handle well the situation when we try to resolve a user that is not present in passwd file.

The easiest solution is to add root definition in the files.

Reply:
I see, thanks for the clarification